src/main/java/frc/robot/RobotContainer.java

```diff
@@ -72,6 +76,15 @@ public RobotContainer() {
         () -> MathUtil.applyDeadband(-m_driverController.getRightX(), OIConstants.kDriverControllerDeadband),
         m_robotDrive));
     
+   m_arm.setDefaultCommand(new RunCommand(() -> m_arm.setSprocketSpeed(
+    MathUtil.applyDeadband(-m_operatorController.getLeftY() / 3, OIConstants.kOperatorControllerDeadband)
```

Comment:
I would slightly prefer doing a * 0.3 instead of /3

Reply:
Should the 30% / third be pulled out into a variable to make it more explicit (and easier to modify in the future) or is it likely to be fixed forever? Right now it reads like a "magic number", which many folks consider a "code smell".

Reply:
I think it could be a variable in Constants.java